content/security/advisory/2019-05-31.adoc

```diff
@@ -61,18 +61,18 @@ issues:
 
 - id: SECURITY-921
   reporter: Jesse Glick, CloudBees, Inc.
-  title: Unsafe Script Security whitelist entry in PLUGIN_NAME
+  title: Unsafe entry in Script Security list of approved signatures in PLUGIN_NAME
   cve: CVE-2019-10328
   cvss:
     severity: High
     vector: CVSS:3.0/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H
   description: |-
-    PLUGIN_NAME provides a custom Script Security whitelist.
+    PLUGIN_NAME provides a custom list of pre-approved signatures for Script Security.
```

Comment:
I'm not sure about the `pre-approved` terminology for these issues. "custom list of approved signatures" also seems ok to me. No strong opinion though.